framework/encode/vulkan_handle_wrappers.h

```diff
@@ -309,11 +308,22 @@ struct PipelineWrapper : public HandleWrapper<VkPipeline>
     // Ray tracing pipeline's shader group handle data
     format::HandleId     device_id{ format::kNullHandleId };
     std::vector<uint8_t> shader_group_handle_data;
+    CreateDependencyInfo deferred_operation;
 
     // TODO: Base pipeline
     // TODO: Pipeline cache
 };
 
+struct DeferredOperationKHRWrapper : public HandleWrapper<VkDeferredOperationKHR>
+{
+    // Record CreateRayTracingPipelinesKHR parameters for safety.
+    HandleUnwrapMemory                             record_handle_unwrap_memory;
```

Comment:
None of the other Wrappers have `record_` in the variable name.  Does `record` mean something special in this case that the other Wrappers don't do?

Reply:
It's not necessary. I can remove it.

Reply:
fixed